packages/react-text/README.md

```diff
@@ -1,5 +1,99 @@
 # @fluentui/react-text
 
-**React Text component and wrappers for [Fluent UI React](https://developer.microsoft.com/en-us/fluentui)**
+> WIP 🚧 - These are not production-ready components as we are still in a Beta phase.
 
-These are not production-ready components and **should never be used in product**. This space is useful for testing new components whose APIs might change before final release.
+<!-- TODO: Add link to the new website -->
+
+The Text component exists to ensure consistency in your application's content by setting fixed sizes and other styles.
+This package also exports wrappers which ensure your text follows the Fluent design standards of typography.
+
+## Usage
+
+To use the Text components in your application, you can start by installing the main package of Fluent UI components:
+
+<!-- TODO: Validate if FluentProvider works without theme. If not, which theme should we refer to -->
+
+```sh
+npm install @fluentui/react-components
+```
+
+```jsx
+import { FluentProvider, Text } from '@fluentui/react-components';
+
+const App = () => (
+  <FluentProvider>
+    <Text>Fluent UI Text!</Text>
+  </FluentProvider>
+);
+```
+
+Or by installing only the `react-text` package. Keep in mind you'll need to install the FluentProvider package as well:
+
+```sh
+npm install @fluentui/react-text
+npm install @fluentui/react-provider
+```
+
+```jsx
+import { FluentProvider } from '@fluentui/react-provider';
```

Comment:
is this what we want? I thought any inner vNext dependency is provided as direct package dependency, thus no need to manually install `react-provider`

Reply:
Does that mean that this can be done then? 
`import { FluentProvider } fron '@fluentui/react-text'`

My understanding was that the package only exports stuff related to the package itself.

Reply:
aah right, no I don't think so (although react-menu uses provider within it's implementation directly). WDYT @ling1726  ?

Reply:
@Hotell `react-menu` should no longer by using provider directly, the one usage is `useFluent` which was introduced before `react-shared-contexts` hosted the same hook. It should actually be changed